lerna.json

```
@@ -1,6 +1,6 @@
 {
-  "packages": ["packages/*", "demo"],
-  "version": "fixed",
+  "packages": ["packages/*"],
+  "version": "independent",
```

```suggestion
  "version": "0.1.8",
```

I think I misunderstood this when we were looking at it. I think we want to tie versions of all packages together, so let's set this to the same version as the gatsby-theme-newrelic package. Lerna will use this as the basis for all future versions.